docs/getting-started.md

```diff
@@ -26,8 +26,8 @@
 
 ## Introduction
 These instructions assume that you have the following accounts setup:
-- Github
-- Slack
+- [Github](https://github.com/login)
+- [Slack](https://slack.com/signin)
```

Comment:
nit: should this link directly to our `thunderbots.slack.com` domain?

Reply:
good idea. updated the link